src/components/UI/Button/Button.tsx

```diff
@@ -1,17 +1,5 @@
-import styled from "@emotion/styled";
 import React, { FunctionComponent, AnchorHTMLAttributes, ButtonHTMLAttributes, LabelHTMLAttributes } from "react";
 
-const ButtonWithSvg = styled.button`
-  width: 40px;
-  height: 40px;
-
-  svg {
-    max-width: 16px;
-    margin-left: auto;
-    margin-right: auto;
-  }
```

Comment:
need this tho if i rmb some cases, svg icons will be passed to children. i saw u move it to global, dunno if there is any other way not